aws/versions.tf

```diff
@@ -0,0 +1,15 @@
+# terraform {
+#   required_providers {
```

Comment:
This file is usually called a `providers.tf` https://docs.sourcegraph.com/dev/background-information/languages/terraform#providers-tf

Reply:
Good callout. I actually commented this out when I was getting some errors when running. Meant to come back later and update the versions so that it worked, but found that it worked perfectly without this file at all. Should I add it back?